documentation/release_notes.rst

```diff
@@ -57,8 +57,10 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
-- Incorrect results may be produced by ``reduce`` and ``transform_reduce`` with 64-bit types and ``std::multiplies``,
-  ``sycl::multiplies`` operations when compiled by Intel® C++ Compiler 2021.3 and newer and executed on GPU devices. 
+- Incorrect results may be produced by ``reduce``, ``reduce_by_segment`` and ``transform_reduce`` with 64-bit types
+  and ``std::multiplies``, ``sycl::multiplies`` operations when compiled by Intel® C++ Compiler 2021.3 and newer
```

Comment:
If I remember correctly, you mentioned that the issue is not limited to multiplication but can be seen with other reduction operations as well. Did I misunderstand?

Reply:
No, you are correct. Fixed.